FinSearchUnified/Bundle/SearchBundleFindologic/FindologicSortingHandlerInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace FinSearchUnified\Bundle\SearchBundleFindologic;
+
+use Shopware\Bundle\SearchBundle\SortingInterface;
+use Shopware\Bundle\StoreFrontBundle\Struct\ShopContextInterface;
+
+interface FindologicSortingHandlerInterface
```

Comment:
Rename this interface to `SortingHandlerInterface`. Its namespace is already mentioning the relation to FINDOLOGIC multiple times.